### ISSUE TITLE
Products

### ISSUE THREAD

**Issue by Marcus084** [Marcus084/Internal#7]
The products have an object:fit effect on it, but when the screen is shrinked it results , they slightly overlap each other, and the user is unable to click the more -info button. This effects the functionality, usability and accessibility considerations the website covers, and I need to fix this issue before due date
